metricflow/engine/metricflow_engine.py

```diff
@@ -66,6 +72,17 @@
 _telemetry_reporter = TelemetryReporter(report_levels_higher_or_equal_to=TelemetryLevel.USAGE)
 _telemetry_reporter.add_python_log_handler()
 
+SIMPLE_DIMENSIONS_WITHOUT_ANY_PROPERTIES: Set[LinkableElementProperty] = {
```

Comment:
Moved to a constant and added the `METRIC` property. This doesn't create changes currently but would be necessary if we added support for metrics in the group by (not just the where filter).